gradio/processing_utils.py

```diff
@@ -347,7 +347,7 @@ def move_files_to_cache(
     postprocess: bool = False,
     check_in_upload_folder=False,
     keep_in_cache=False,
-) -> dict:
+):
```

Comment:
This was incorrect, fixed so that we don't need to ignore the types

Reply:
Thanks! It's better like this indeed :+1: